hip-0000-anonCreds-verifiable-data-registry.md

```diff
@@ -0,0 +1,147 @@
+---
+hip: 0000
+title: AnonCreds Verifiable Data Registry
+author: [Sumabala P Nair](https://github.com/sumapnair-ibm), [Jack Hsu]()
```

Comment:
Our validator checks for names in this form -> Firstname Lastname (@githubname or email)
```suggestion
author: Sumabala P Nair (@sumapnair-ibm), Jack Hsu (@jackgithubname or email)
```

Reply:
@mgarbs When I try to sign off and commit the suggestion about the name, I  see the button is greyed out and github says 'Outdated suggestion cannot be applied' Please advise.